orchestration/dagster_orchestration/hca_orchestration/tests/e2e/conftest.py

```diff
@@ -0,0 +1,114 @@
+import logging
+from typing import Iterable
+import uuid
+
+import pytest
+from dagster_utils.contrib.google import authorized_session
+from google.cloud.bigquery.client import Client
+
+from hca_manage.common import data_repo_host, get_api_client
+from hca_manage.dataset import DatasetManager
+
+MONSTER_TEST_DATASET_SENTINEL = "MONSTER_TEST_DELETEME"
+
+
+@pytest.fixture
+def tdr_bigquery_client():
+    return Client(_http=authorized_session())
+
+
+@pytest.fixture
+def delete_dataset_on_exit():
+    return True
+
+
+@pytest.fixture
+def existing_dataset_id():
+    return None
```

Comment:
This is a hacky way to force the test to resume from a previously ingested dataset. I'd like to parameterize this via a CLI arg.